### PR TITLE
Customer tests

### PR DESCRIPTION
**Pull request by ViktoriaMilkevich**
Add facade for addRestaurant and sign in tests for customer

### CODE REVIEW

File: src/test/java/com/easyrest/tests/customer/SignInTest.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        signInFacade = new SignInFacade(driver);
+        wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+        signInPage.goToSignInPage();
+    }
+
+    @Test
+    public void logInWithValidData() {
+        signInFacade.signIn(clientEmail, clientPassword);
+        wait.until(ExpectedConditions.urlToBe(expectedUrl));
+
```
Comment (Zeicks) [inline src/test/java/com/easyrest/tests/customer/SignInTest.java]: Why empty lines?

File: src/test/java/com/easyrest/tests/customer/SignInTest.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import java.time.Duration;
+
+
+public class SignInTest extends BaseTest {
+
+    private SignInFacade signInFacade;
+    private SignInPage signInPage;
+    private final String clientEmail = ConfigProvider.clientEmail;
+    private final String clientPassword = ConfigProvider.clientPassword;
+    private final String fakePassword =ConfigProvider.fakePassword;
```
Comment (vadymtsehelny) [inline src/test/java/com/easyrest/tests/customer/SignInTest.java:24]: Code formatting

File: src/main/java/com/easyrest/facade/AddRestaurantFacade.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
+package com.easyrest.facade;
+
+import com.easyrest.components.AddRestaurant;
+import com.easyrest.components.customer.Right_sideMenu;
+import com.easyrest.pages.customerPanel.MyRestaurants;
+import org.openqa.selenium.WebDriver;
+
+public class AddRestaurantFacade {
+    private WebDriver driver;
```
Comment (martadiachok) [inline src/main/java/com/easyrest/facade/AddRestaurantFacade.java]: i think driver is not necessary here, because you do not use it in that class

**Review by Zeicks — APPROVED** [review]

File: src/test/java/com/easyrest/tests/customer/SignInTest.java (hunk 2 above)
Comment (vadymtsehelny) [reply src/test/java/com/easyrest/tests/customer/SignInTest.java:21]: You did Rearrange code

**Review by vadymtsehelny — APPROVED** [review]

**Review by martadiachok — APPROVED** [review]

**Review by dmatsev — APPROVED** [review]

**Pull request merged**